### DISCUSSION TITLE
How to debug react-hook-form in WebStorm?

### DISCUSSION THREAD

**Discussion by anhquande** [react-hook-form/react-hook-form#2608] [Q&A] [answered]
Hello guys,

I am using WebStorm as IDE, and developing a Reactjs application using react-hook-form.
In order to debug react-hook-form, I have to copy the source code of react-hook-form and place the source code inside the  folder /src of my project.

It is inconvenient because I have to change all the `import` of react-hook-form every time I want to debug it.

Is there any other way to do it better?

Thanks

**Comment by bluebill1049** [comment, marked as answer]
we do supply with source map for this lib, i used to use webstorm debugger as well, but i moved on with chrome devtool.
